### ISSUE TITLE
Failed to reproduce Campus dataset results

### ISSUE THREAD

**Issue by jpsml** [sail-sg/mvp#6]
Hi, I have changed the mvp_campus.yaml file for setting END_EPOCH to 40 in order to try to reproduce the results reported in the paper. However, I got the results below:

     | Actor 1 | Actor 2 | Actor 3 | Average | 
 PCP |  91.02  |  93.02  |  88.48  |  90.84  |	 Recall@500mm: 1.0000

Should I change anything else in order to reproduce the paper results for the Campus dataset? Thanks.

**Comment by wqyin** [comment]
Hello, I am also having some problems reproducing Campus dataset results, the pre-trained weight did not work either. I am getting a very similar accuracy: average 90.8%. Did you figure out how to modify the training parameters to get it to work on the Campus dataset?

**Comment by jpsml** [comment]
No, I was not able to reproduce the paper results.
